app.py

```diff
@@ -104,6 +101,7 @@ def create_book():
             image_url = params.get('image_url')
             isbn = params.get('isbn')
             published_date = params.get('published_date')
+            shelf = ShelfModel.query.filter_by(id=2).first()
```

Comment:
Cool, good to hardcode this in for now for our single user. In the future, we could change this to look through the user's shelves and return the one with the `already_read` type.